docs/src/man/doctests.md

```diff
@@ -263,6 +263,12 @@ julia> @time [1,2,3,4]
     The global filters, filters defined in `@meta` blocks, and filters defined with the `filter`
     keyword argument are all applied to each doctest.
 
+## Fixing outdated Doctests
+
+To fix outdated doctests, the `doctest` flag to [`makedocs`](@ref) can be set to
+`doctest = :fixup`. This will run the doctests, and overwrite the old results with
+the new output.
+
```

Comment:
Might be good to recommend here that the user should `git commit` their code changes, just in case, before running the doc build with this option on. That way, should the find-and-replace algorithm go a bit wonky and completely mess up the source file, the user can easily just `git reset --hard` to their state.